app/design/frontend/Clothes/touch/Magento_Cms/layout/cms_index_index.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="main.content">
+            <block class="Magento\Framework\View\Element\Template" name="mag-15.one" template="Magento_Cms::default/mag15.phtml">
+                <arguments>
+                    <argument name="argumentClassFirst" xsi:type="string">-argument-first</argument>
```

Comment:
@yuratouch Try to improve your code, using 2 arguments instead of 4 (1 argument for 2 different classes and 1 argument for 2 different contents)